packages/splits-sdk/package.json

```diff
@@ -1,37 +1,37 @@
 {
   "name": "@0xsplits/splits-sdk",
-  "version": "4.0.0-alpha.4",
+  "version": "4.0.2-alpha.10",
   "description": "SDK for the 0xSplits protocol",
-  "main": "./dist/index.js",
-  "typings": "./dist/index.d.ts",
+  "main": "./dist/src/index.js",
+  "typings": "./dist/src/index.d.ts",
```

Comment:
do you know why this change was needed? seems like maybe something changed with the lerna build or something?

Reply:
yeah I am not sure why this happened but yeah I realized lerna was building stuff out to src so corrected this